hexa/pipelines/models.py

```diff
@@ -323,6 +328,15 @@ def get_token(self):
             }
         )
 
+    def merge_pipeline_config(
+        self,
+        config: typing.Mapping[typing.Dict, typing.Any],
+        pipeline_version_config: typing.Mapping[typing.Dict, typing.Any],
+    ):
+        provided_config = config if config else self.config
+        merged_config = {**pipeline_version_config, **provided_config}
```

Comment:
I'm wondering if we should only takes the keys where their value is != than None?